tuxemon/core/item.py

```diff
@@ -145,6 +147,33 @@ def advance_round(self):
         """
         return
 
+    def validate(self, target):
+        """Ensures that the target meets all conditions that the
+        item has on it's use.
+
+        :param user: The monster or object that is using this item.
```

Comment:
I don't see the 'user' parameter

Reply:
Please update the docstring by removing unused parameters.

Reply:
I resolved it because I had fixed it locally, I just hadn't pushed it up yet.
There's no need to re-review a PR if no changes have been pushed up.